src/block/block.js

```diff
@@ -146,9 +149,6 @@ function Block({
   );
 }
 
-Block.defaultProps = {
-  overrides: {},
-  as: 'div',
-};
-
-export default Block;
+const BlockComponent = React.forwardRef<BlockPropsT, HTMLElement>(Block);
+BlockComponent.displayName = 'Block';
```

Comment:
I think the styletron babel plugin adds the displaynames - should we remove this ?

Reply:
it's a `forwardRef` component not `styled` =)

Reply:
ah, I've missed that, thanks!